app/test/frontend/golden/search_page.html

```diff
@@ -78,9 +78,10 @@ <h1 class="_visuallyhidden">Dart pub</h1>
   
 <div class="listing-sort-header">
   <div class="tooltip-base">
-    <span class="tooltip-dotted">Sorted by:</span> relevance
+    <span class="tooltip-dotted">Sorted by:</span>
+    <span id="sort-control" data-sort="top">overall score</span>
     <div class="tooltip-content tooltip-bottom-left">
-      Packages are sorted by the combination of text match and overall score. More information on <a href="/help#ranking">ranking</a>.
+      Packages are sorted by the overall score. More information on <a href="/help#ranking">ranking</a>.
```

Comment:
The search page *should* be ordered by text match, right?

Reply:
normally yes, but in the template test it is rendered with another ordering